dev-util/pycharm-community/pycharm_community_bin-2020.1.1.recipe

```diff
@@ -1,18 +1,19 @@
 SUMMARY="A Lightweight IDE for Python & Scientific development"
 DESCRIPTION="PyCharm is a Python IDE for professional developers by JetBrains"
 HOMEPAGE="https://www.jetbrains.com/pycharm/"
-COPYRIGHT="2018 JetBrains s.r.o."
+COPYRIGHT="2020 JetBrains s.r.o."
 LICENSE="Apache v2"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="https://download.jetbrains.com/python/pycharm-community-$portVersion.tar.gz#noarchive"
-CHECKSUM_SHA256="8eedae360aaf3e0480171d43a12db66f0623b5a7cc1cf239e98e8d9aed272aaa"
+CHECKSUM_SHA256="715a6a1fc1638078a36b5b45e36e3dbe1ce097aa974555023b4b3d957d00585c"
 ADDITIONAL_FILES="PyCharm_Logo.hvif"
 
 ARCHITECTURES="any"
 DISABLE_SOURCE_PACKAGE="true"
 
 PROVIDES="
 	pycharm_community_bin = $portVersion
+	pycharm = $portVersion
```

Comment:
Why add this provides?

Reply:
I wanted to have a common provide for `pycharm_community_bin` and `pycharm_professional_bin`. If it's not a correct usage, I'll remove it.

Reply:
usually we add "app:pycharm"